sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,654 @@
+"""Stacking classifier and regressor."""
+
+# Authors: Guillaume Lemaitre <g.lemaitre58@gmail.com>
+# License: BSD 3 clause
+
+from abc import ABCMeta, abstractmethod
+from copy import deepcopy
+
+import numpy as np
+from joblib import Parallel, delayed
+
+from ..base import clone
+from ..base import ClassifierMixin, RegressorMixin, TransformerMixin
+from ..base import is_classifier, is_regressor
+from ..base import MetaEstimatorMixin
+
+from .base import _parallel_fit_estimator
+
+from ..linear_model import LogisticRegression
+from ..linear_model import LinearRegression
+
+from ..model_selection import cross_val_predict
+from ..model_selection import check_cv
+
+from ..preprocessing import LabelEncoder
+
+from ..utils import Bunch
+from ..utils.metaestimators import _BaseComposition
+from ..utils.metaestimators import if_delegate_has_method
+from ..utils.multiclass import check_classification_targets
+from ..utils.validation import check_is_fitted
+from ..utils.validation import column_or_1d
+
+
+class _BaseStacking(_BaseComposition, MetaEstimatorMixin, TransformerMixin,
+                    metaclass=ABCMeta):
+    """Base class for stacking method.
+
+    Warning: This class should not be used directly. Use derived classes
```

Comment:
Seems superfluous given that it's private and abstract. good luck instantiating it.